README.md

```diff
@@ -23,6 +23,7 @@ __Windows__
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (optional, required only for LLVM build; add to PATH)
 * [Python](https://www.python.org/downloads/) (optional, required only for LLVM build; add to PATH)
+* [Git] (https://git-scm.com/downloads) (Select "use Git from the Windows Command Prompt")
```

Comment:
@paulsapps But everyone does I would point out, absolutely everything the other project that exists is just the minimum to say.
If you not tell someone not know git, does not understand.
And come on github does not mean Git know, nothing to do.

And your Git command, I not even know me.
We always use "git clone"